core/src/main/java/com/novoda/merlin/MerlinBuilder.java

```diff
@@ -141,7 +138,7 @@ public Merlin build(Context context) {
                 responseCodeValidator
         );
 
-        Registerer merlinRegisterer = new Registerer(connectableRegisterer, disconnectableRegisterer, bindableRegisterer);
+        Registerer merlinRegisterer = new Registerer(connectables, disconnectables, bindables);
```

Comment:
We now have a `Registerer` and individual `Register`s for each of the `Registerable` components, rather than implying that you register directly on the `MerlinRegisterer`.